docs/guide/content_management.md

```diff
@@ -30,13 +30,13 @@ Under this root there are preset top level Locations in each installation which
 
 ![Content and Media top Locations](img/content_structure_media_library.png)
 
-#### Content
+#### Home
 
-**Content** is the top level Location for the actual contents of a site.
+**Home** is the top level Location for the actual contents of a site.
 This part of the tree is typically used for organizing folders, articles, information pages, etc.
 This means that it contains the actual content structure of the site,
 which can be viewed by selecting the **Content structure** tab in the Content mode interface.
-The default ID number of the **Content** Location is 2, but it can be [modified via configuration](config_repository.md#top-level-locations).
+The default ID number of the **Home** Location is 2, but it can be [modified via configuration](config_repository.md#top-level-locations).
```

Comment:
I think we should mention here that this is for back-office / AdminUI purpose only. Repository has no knowledge of "Content structure" Location configuration (maybe except for SiteAccess tree root).

Reply:
Added clarification in the link's target page.